src/Verify/VirtualizedRunHelper.cs

```diff
@@ -1,9 +1,11 @@
 class VirtualizedRunHelper
 {
     // e.g. WSL or docker run (https://github.com/VerifyTests/Verify#unit-testing-inside-virtualized-environment)
-    bool appearsToBeLocalVirtualizedRun;
-    string originalCodeBaseRootAbsolute = string.Empty;
-    string mappedCodeBaseRootAbsolute = string.Empty;
+    internal bool AppearsToBeLocalVirtualizedRun { get; }
+
+    string _originalCodeBaseRootAbsolute = string.Empty;
+    string _mappedCodeBaseRootAbsolute = string.Empty;
```

Comment:
while the fields prefix doesn't seem to be explicitly disallowed (https://github.com/VerifyTests/Verify/blob/main/src/.editorconfig#L22-L25), it doesn't seem to be used convention in this project.

Reply:
Yes I hesitated, will use `this.` then.